NYTimesMostViewedArticles/ArticlesList/NYTimesMostViewedArticleViewController.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  NYTimesMostViewedArticleVC.swift
+//  NYTimesMostViewedArticles
+//
+//  Copyright © 2020 Sagar Pahwa. All rights reserved.
+//
+
+import UIKit
+
+extension UIViewController: UIViewControllerType {
+    func push(vc: UIViewControllerType, animated: Bool) {
+        if let vc = vc as? UIViewController {
+            navigationController?.pushViewController(vc, animated: animated)
+        }
+    }
+}
+
+class NYTimesMostViewedArticleViewController: UIViewController {
+    @IBOutlet weak var tableView: UITableView!
+    
+    @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
+    
+    let cellNibName = "ViewedArticleCell"
+    
+    lazy var viewModel: NYTimesMostViewedArticleViewModel = {
+        return NYTimesMostViewedArticleViewModel(api: MostPopularArticlesNetworkService(client: NetworkClient()))
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+        configureUI()
+        fetchMostViewedArticles()
+    }
+    
+    func configureUI() {
+        tableView.register(UINib(nibName: cellNibName, bundle: Bundle.init(for: ViewedArticleCell.self)), forCellReuseIdentifier: cellNibName)
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.isAccessibilityElement = true
```

Comment:
All these cab be set from stroyboard itself.

Reply:
set delegates and datasource in storyboard except couldnt find the accessibility option for tableView hence leaving it programmatically